index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>my-portfolio-project</title>
+  <link rel="stylesheet" href="style.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Andika&family=Inter:wght@300;400;500;700&family=Poppins:ital,wght@0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,200&family=Roboto:wght@100;300;400;500;700;900&family=Ubuntu:ital,wght@1,300&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header class="container">
+  <!-- start of nav section-->
+  <nav class="nav-wrapper">
+    <!--nav left side-->
+    <div class="nav-left">
+      <span>My Logo</span>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+      <div class="nav-list">
+        <ul>
+          <li><a href="#" class="nav-link">Portfolio</a></li>
+          <li><a href="#" class="nav-link">About</a></li>
+          <li><a href="#"  class="nav-link">Contact</a></li>
+        </ul>
+        </div>
```

Comment:
- [ ] Please **remove** this block because it is not part of the project requirements. Also, this block will be implemented later along with new requirements.